isIsogram.js

```diff
@@ -0,0 +1,13 @@
+export function isIsogram(str) {
+    for (let i = 0; i < str.length; i++) {
+
+        const strNoCase = str.toLowerCase();
+
+        for (let j = i + 1; j < str.length; j++) {
+            if (strNoCase.charAt(i) == strNoCase.charAt(j)) {
```

Comment:
"Double equal" will work but is potentially dangerous. Read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness